packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1430,6 +1430,15 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     ): readonly GridMouseHandler[] => [...mouseHandlers, ...this.mouseHandlers]
   );
 
+  getCachedRenderer = memoize(
+    (rendererProp?: IrisGridRenderer) => rendererProp ?? new IrisGridRenderer()
+  );
```

Comment:
Should this have `{ max: 1 }`? No need to keep other renderers around if it changed (even though it should be uncommon)